org.ekstep.lessonbrowser-1.4/editor/plugin.js

```diff
@@ -66,25 +59,5 @@ org.ekstep.collectioneditor.basePlugin.extend({
             className: 'ngdialog-theme-plain lessonbrowser-dialog'
         });
     },
-
-    getEkstepRepo: function() {
```

Comment:
Can you share more details.
* Who added this method?
* Is anyone using this?
* Why are you removing this?